### PR TITLE
Spicing up the frontend

### PR DESCRIPTION
**Pull request by sloan-seamus**
Creating a new claim in New Claim will redirect to View Claims instead of Edit Page --Until loading issue is resolved--

Modified color scheme --No more "bruised" look
Rearranged header toolbar --Different menu to come soon

### CODE REVIEW

File: frontend/src/app/header/header.component.html [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <header class="head nav-bar">
     <ly-toolbar position="relative" bg="primary" [elevation]="5" class="auto-height">
-      <img src="../../assets/ford-logo.png" alt="Logo" class="center-to-page width-100 homepage-logo" routerLink="/home">
+      <img src="../../assets/ford-logo.png" alt="Logo" class="center-to-page width-100 homepage-logo vspacing" routerLink="/home">
```
Comment (sloan-seamus) [inline frontend/src/app/header/header.component.html]: I need a little help with this. I think I'm stupid.

I want a little bit of spacing between the top/bottom of the Ford logo and the toolbar, but padding and margins don't seem to do anything??

Comment (ej3076) [reply frontend/src/app/header/header.component.html]: @sloan-seamus you're adding a class called `vspacing` but that's not defined anywhere...

use the existing margin classes - if you want more space above / below on the toolbar add these: `mtm mbm`
margin-top-medium, margin-bottom-medium, these are defined in styles.scss

File: frontend/src/app/home/home.component.html [hunk 2]
```diff
@@ -2,13 +2,20 @@
...
-    ly-button
-    raised
-    bg="accent"
-    lyTyp="display2"
-    class="width-100 mth mbh">
-    Start a New Claim
-  </a>
+
+  <button ly-button raised outlined color="primary" size="large" lyType="title" class="width-100 mth mbh" routerLink="/claims/new"> Start a New Claim </button>
+<!--
```
Comment (ej3076) [inline frontend/src/app/home/home.component.html]: Remove all these too

File: frontend/src/app/middleware/middleware.service.ts [hunk 3]
```diff
@@ -83,7 +83,7 @@ export class MiddlewareService {
       .subscribe(
         success => {
           // Log when successful / errored navigation
-          this.router.navigate([`/claims/${claim.vehicle.vin}`]).then(
+          this.router.navigate([`/claims`]).then(
```
Comment (ej3076) [inline frontend/src/app/middleware/middleware.service.ts]: You can probably undo this as well - in my PR i have a little dumb workaround for the data not showing.

File: frontend/src/app/view-claims/view-claims.component.html [hunk 4]
```diff
@@ -4,14 +4,14 @@
     <!--STATUS # 0-->
   <ly-tab>
     <button ly-tab-label>Awaiting Settlement Option</button>
-    <ly-grid *ngIf="sortClaims(0).length > 0; else noClaims" container [spacing]="16">
+    <ly-grid *ngIf="sortClaims(0).length > 0; else hasClaims" container [spacing]="16">
```
Comment (ej3076) [inline frontend/src/app/view-claims/view-claims.component.html]: This looks like you check if there is any claims, and if not... go to `hasClaims`? What is this doing?

File: frontend/src/app/view-claims/view-claims.component.html [hunk 5]
```diff
@@ -49,7 +49,7 @@ <h3 [lyTyp]="'title'">{{ claim.vehicle.vin }}</h3>
   <!--STATUS # 3-->
   <ly-tab>
     <button ly-tab-label (click)="sortClaims(3)">Claim Resolved</button>
-    <ly-grid *ngIf="sortClaims(3).length > 0; else noClaims" container [spacing]="16">
+    <ly-grid *ngIf="sortClaims(3).length > 0; then hasClaims else noClaims" container [spacing]="16">
```
Comment (ej3076) [inline frontend/src/app/view-claims/view-claims.component.html]: Same here - what is this doing?

**Review by ej3076 — CHANGES_REQUESTED** [review]
Just a few comments.

**Review by dsifford — APPROVED** [review]
Nothing further to add.

File: frontend/src/app/view-claims/view-claims.component.html (hunk 5 above)
Comment (sloan-seamus) [reply frontend/src/app/view-claims/view-claims.component.html]: That was something I was trying out and forgot to delete

File: frontend/src/app/middleware/middleware.service.ts (hunk 3 above)
Comment (sloan-seamus) [reply frontend/src/app/middleware/middleware.service.ts]: I left it for the time being so my branch is still functional

File: frontend/src/app/header/header.component.html [hunk 6]
```diff
@@ -1,24 +1,30 @@
-<header class="head nav-bar">
-    <ly-toolbar position="relative" bg="primary" [elevation]="5" class="auto-height">
-      <img src="../../assets/ford-logo.png" alt="Logo" class="center-to-page width-100 homepage-logo" routerLink="/home">
+
+<header class="head mbm">
+    <ly-toolbar position="relative" bg="primary" [elevation]="5" class="header-height">
+      <a href="/home">
```
Comment (ej3076) [inline frontend/src/app/header/header.component.html]: Change this from `href` to `routerLink`

File: frontend/src/app/middleware/middleware.service.ts (hunk 3 above)
Comment (ej3076) [reply frontend/src/app/middleware/middleware.service.ts]: Revert this - my branch fixes it temporarily

**Review by ej3076 — CHANGES_REQUESTED** [review]
Couple of changes

**Review by ej3076 — APPROVED** [review]
Looks good

**Pull request merged**